vignettes/deploy-to-aks/deploy-to-aks.Rmd

```diff
@@ -0,0 +1,144 @@
+---
+title: "Deploying to AKS with Azure ML SDL for R"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Deploying to AKS with Azure ML SDL for R}
+  %\VignetteEngine{knitr::rmarkdown}
+  \use_package{UTF-8}
+---
+
+This article demonstrates how to deploy a model and conduct inferencing in the model using AzureML SDK for R. We will take a previous saved model. The saved model was trained with the Iris Dataset and can be used to determine if a flower is one of three Iris flower species (setosa, versicolor, virginica).
```

Comment:
Can you add a link to whichever sample or vignette has the training code for the model you're using